src/Tribe/Editor/Configuration.php

```diff
@@ -8,6 +8,18 @@
  * @since 4.8
  */
 class Tribe__Editor__Configuration implements Tribe__Editor__Configuration_Interface {
+	/**
+	 * Convert HTML entities in country names to their corresponding characters 
+	 *
+	 * @param array $countries The array of countries
+	 * @return array
```

Comment:
```suggestion
	 * Convert HTML entities in country names to their corresponding characters.
	 *
	 * @since TBD
	 *
	 * @param array $countries The list of countries.
	 *
	 * @return array The list of countries.
```